authority/authorize.go

```diff
@@ -130,22 +130,19 @@ func (a *Authority) AuthorizeAdminToken(r *http.Request, token string) (*linkedc
 	// According to "rfc7519 JSON Web Token" acceptable skew should be no
 	// more than a few minutes.
 	if err := claims.ValidateWithLeeway(jose.Expected{
-		Issuer: prov.GetName(),
+		Issuer: "step-admin-client/1.0",
```

Comment:
So, if I understand correctly, any client wishing to integrate with the admin API needs to create a JWT with `step-admin-client/1.0` as the `iss` claim after this change? I'm not sure if it's a hard requirement to restrict it to this, but it makes sense to do so for validation purposes. I'm thinking about use cases in which someone doesn't use our CLI to interact with the admin API, which might not happen all too often, but is not unexpected.

Reply:
With ad5aedfa I fixed the keep backward compatibility issue.